ddp-workspace/projects/ddp-singular/src/app/components/pages/participant-list/participant-list.component.ts

```diff
@@ -403,7 +410,7 @@ export class ParticipantsListComponent implements OnInit {
 
   private hasOnlyAddParticipantActivity(activities: ActivityInstance[]): boolean {
     return (
-      activities.length === 1 &&
+      !!activities && activities.length === 1 &&
```

Comment:
**#minor #optional**

`!!activities && activities.length === 1` 
can be shortened like
`activities?.length === 1`